### ISSUE TITLE
Using with screenreader

### ISSUE THREAD

**Issue by burmancomp** [lantongxue/rustdesk-api-server-pro#39]
I have (I hope) successfully installed rustdesk-api-server-pro. I have added user with admin privileges from CLI, and I guess I should be able to login to the management panel from web interface. I can open that page but I cannot login. Screenreader (NVDA which is open source; I guess that Orca would give somewhat similar results but I have no possibility to test with it) tells there are some unlabelled buttons, edit field, password field and another edit field. Jaws (commercial screenreader) tells that there is image with changing content - and I might guess it is captcha. But I cannot figure out what I actually should enter to fields to login. Of course it it is possible that my installation is not so successful as I think but I have not found any issues from log files.

I think this issue is something to do with #35 . I tried also to set hcaptcha accessibility cookie but without effect (maybe it depends on browser security settings or similar how well it works).

It would be helpful to have information what is meaning of all fields/buttons (there are also at least one check box), and if you tab between them with tab/shift tab key what is their order (where you are when you press tab key repeatedly).

Thank you for any help.
